app/Http/Controllers/ExampleController.php

```diff
@@ -6,5 +6,26 @@
 
 class ExampleController extends Controller
 {
-    
+    function login(){
+        return view('login');
+    }
+
+    function cv(){
+        return view('cv');
+    }
+
+    function contactus(){
+        return view('contactus');
+    }
+
+    function personaldata(Request $request){
+        $name= $request->name;
```

Comment:
nice work ya Ellen,,

your code is very neat

Reply:
ok thanks